airbyte-cdk/python/airbyte_cdk/sources/declarative/parsers/model_to_component_factory.py

```diff
@@ -175,7 +174,6 @@ def _init_mappings(self) -> None:
             BasicHttpAuthenticatorModel: self.create_basic_http_authenticator,
             BearerAuthenticatorModel: self.create_bearer_authenticator,
             CheckStreamModel: self.create_check_stream,
-            CompositeErrorHandlerModel: self.create_composite_error_handler,
```

Comment:
Can we really do that without migration? I fear that it could break thousands of connectors... In order words, I think we can remove the code but not the YAML model

Reply:
Speaking with @girarda, we thought this was a big breaking change, and considered this is not suitable for a tech debt day. Will move to another task.

Reply:
That makes sense to me ; it indeed feel very big. Can we still aim to remove the CompositeErrorHandler code but not the low-code interface? This would require us to keep the `CompositeErrorHandlerModel: self.create_composite_error_handler,` line in the model_to_component_factory but to instantiate another class instead. That seems feasible, right?

Reply:
Reverted some of the changes, and pointed CompositeErrorHandler: to defualt_error_handler. Some code gets re-added when running `poetry run poe build`. Are these changes what you meant @maxi297 ?

Reply:
Somewhat yes. However, there seems to be missing some work to make it non breaking as [source-zendesk-support is failing with the current changes](https://storage.cloud.google.com/airbyte-ci-reports-multi/airbyte-ci/connectors/test/pull_request/strosek_td-issue-8019/1723233874/d31a10b36532f72e7f623de3a46d8f3e45d29a48/source-zendesk-support/2.7.1/output.html)